lms/templates/dashboard.html

```diff
@@ -206,7 +206,13 @@
               %>
               <%include file='dashboard/_dashboard_course_listing.html' args='course_overview=course_overview, course_card_index=dashboard_index, enrollment=enrollment, is_unfulfilled_entitlement=is_unfulfilled_entitlement, is_fulfilled_entitlement=is_fulfilled_entitlement, entitlement=entitlement, entitlement_session=entitlement_session, entitlement_available_sessions=entitlement_available_sessions, entitlement_expiration_date=entitlement_expiration_date, entitlement_expired_at=entitlement_expired_at, show_courseware_link=show_courseware_link, cert_status=cert_status, can_refund_entitlement=can_refund_entitlement, can_unenroll=can_unenroll, credit_status=credit_status, show_email_settings=show_email_settings, course_mode_info=course_mode_info, is_paid_course=is_paid_course, is_course_blocked=is_course_blocked, verification_status=course_verification_status, course_requirements=course_requirements, dashboard_index=dashboard_index, share_settings=share_settings, user=user, related_programs=related_programs, display_course_modes_on_dashboard=display_course_modes_on_dashboard, show_consent_link=show_consent_link, enterprise_customer_name=enterprise_customer_name, resume_button_url=resume_button_url' />
             % endfor
-
+            % if not load_all_courses and courses_count < total_enrollments:
+                <br/>
+                 ${courses_count} ${_("results successfully populated,")}
+                 <a href="${reverse('dashboard')}?load_all_course">
+                     ${_("click to load all enrolled courses")}
```

Comment:
@marcotuts: Could you please provide the copy for the text in this situation? (Where we're loading only your most recent N enrollments, but give you the option to load the rest.)

Reply:
Thanks for checking this, I’m ok with the text as shown above. (This time I will click add reply to add the message :))

Reply:
Thanks for checking this, I’m ok with the text as shown above. (This time I will click add reply to add the message :))

Reply:
Edit: "Click to load all enrolled courses" <--- shift to capitalized first word.